apps/admin/backend/src/app.tally_report_data.test.ts

```diff
@@ -35,6 +45,13 @@ beforeEach(() => {
   featureFlagMock.enableFeatureFlag(
     BooleanEnvironmentVariableName.SKIP_CAST_VOTE_RECORDS_AUTHENTICATION
   );
+  mockOf(initializeGetWorkspaceDiskSpaceSummary).mockReturnValue(() =>
```

Comment:
unrelated, but this test kept flaking locally, so copied this pattern from other tests that depend on this function to de-flake